guides/common/modules/proc_creating-partition-tables.adoc

```diff
@@ -15,7 +15,7 @@ To use the CLI instead of the {ProjectWebUI}, see the xref:cli-creating-partitio
 . Select the *Default* checkbox if you want to set the template to automatically associate with new organizations or locations.
 . Select the *Snippet* checkbox if you want to identify the template as a reusable snippet for other partition tables.
 . From the *Operating System Family* list, select the distribution or family of the partitioning layout.
-For example, {RHEL}, CentOS, and Fedora are in the Red Hat family.
+For example, {EL}, CentOS, and Fedora are in the Red Hat family.
```

Comment:
I'd say this should stay the way it was. It's listing concrete examples and `Enterprise Linux` doesn't really exist.
```suggestion
For example, {RHEL}, CentOS, and Fedora are in the Red Hat family.
```

Or you can use
```suggestion
For example, {EL} ({RHEL}, CentOS, Rocky, etc) and Fedora are in the Red Hat family.
```

But that's quite long.

Reply:
Reverted!